apps/ai-image-generator/frontend/src/configs/launch-darkly/launchDarklyConfig.ts

```diff
@@ -1,5 +1,6 @@
 const launchDarklyConfig = {
-  clientSideID: `${process.env.REACT_LAUNCHDARKLY_CLIENT_ID}`,
+  // TO DO: Add as env variable
+  clientSideID: '588a047044b03e0b3211298e',
```

Comment:
we probably want to undo this for the PR - Here's the ticket to fix it https://contentful.atlassian.net/browse/INTEG-1070

Reply:
good call! just updated